docs/xxidletuningcompactonidle.md

```diff
@@ -26,23 +26,23 @@
 
 **(Linux&reg; only)**
 
-This option controls garbage collection processing with compaction when the status of the OpenJ9 VM is set to idle.
+This option controls garbage collection processing with compaction when the OpenJ9 VM is running in a container and the state of the VM is set to idle.
 
-<i class="fa fa-exclamation-triangle" aria-hidden="true"></i> **Restrictions:** This option applies only to Linux architectures when the Generational Concurrent (`gencon`) garbage collection policy is in use. This option is not effective if the object heap is configured to use large pages.
+<i class="fa fa-exclamation-triangle" aria-hidden="true"></i> **Restrictions:** This option applies only to Linux&reg; architectures when the Generational Concurrent (`gencon`) garbage collection policy is in use. This option is not effective if the object heap is configured to use large pages.
 
 ## Syntax
 
         -XX:[+|-]IdleTuningCompactOnIdle
 
 | Setting                        | Effect  | Default                                                                            |
 |--------------------------------|---------|:----------------------------------------------------------------------------------:|
-| `-XX:+IdleTuningCompactOnIdle` | Enable  |                                                                                    |
-| `-XX:-IdleTuningCompactOnIdle` | Disable | <i class="fa fa-check" aria-hidden="true"></i><span class="sr-only">yes</span> |
+| `-XX:+IdleTuningCompactOnIdle` | Enable  |  <i class="fa fa-check" aria-hidden="true"></i><span class="sr-only">yes</span>    |
+| `-XX:-IdleTuningCompactOnIdle` | Disable |                                                                                    |
 
 
-If you enable this option, the OpenJ9 VM attempts to compact the object heap when the VM status is idle.
+If the OpenJ9 VM is running in a container and the state is set to idle, the VM attempts to compact the object heap following a garbage collection cycle. The garbage collection cycle is controlled by the `-XX:+IdleTuningGcOnIdle` option, which is also enabled by default when the OpenJ9 VM is running inside a container.
```

Comment:
Again, this behavior is not specific to container env, so the statement `If the OpenJ9 VM is running in a container` can be avoided.